jenkins/internal-ci/generic/release.groovy

```diff
@@ -0,0 +1,311 @@
+                        sh build_release_info.sh -b $branch -v $version -l $integration_dir/$release_tag/prod -t $cortx_build_dir/$release_tag/3rd_party
+                        sh build_readme.sh $integration_dir/$release_tag
+                    popd
+                    
+                    cp $integration_dir/$release_tag/README.txt .
+                    cp $integration_dir/$release_tag/dev/RELEASE.INFO .
+                    
+                """
+                sh label: 'Generate Changelog', script: """
+                    pushd scripts/release_support
+                        sh +x changelog.sh ${currentBuild.previousBuild.getNumber()} ${currentBuild.number} ${ARTIFACT_LOCATION}
+                    popd
+                    cp /root/git_build_checkin_stats/clone/git-build-checkin-report.txt CHANGESET.txt 
+                    cp CHANGESET.txt $integration_dir/$release_tag/dev
+                    cp CHANGESET.txt $integration_dir/$release_tag/prod
+                """
+            }
+        }
+        
+        stage ('Additional Files') {
+            steps {
+
+                sh label: 'Additional Files', script:'''
+                mkdir -p $cortx_build_dir/$release_tag/cortx_iso
+                mv $integration_dir/$release_tag/prod/* $cortx_build_dir/$release_tag/cortx_iso
+                mkdir -p $integration_dir/$release_tag/prod/iso
+                mv $cortx_build_dir/$release_tag/* $integration_dir/$release_tag/prod
+                cp $integration_dir/$release_tag/prod/*/*.INFO $integration_dir/$release_tag/prod
+                        
+                rm -rf "$cortx_build_dir/$release_tag"
+                '''
+            }
+        }
+
+       stage ('Tag last_successful') {
+            steps {
+                script { build_stage = env.STAGE_NAME }
+                sh label: 'Tag last_successful', script: '''
+                    pushd $integration_dir
+                    test -L last_successful && rm -f last_successful
+                    test -L last_successful_prod && rm -f last_successful_prod
+                    ln -s $integration_dir/$release_tag/dev last_successful
+                    ln -s $integration_dir/$release_tag/prod last_successful_prod
+                    popd
+                '''
+            }
+        }
+
+        stage ("Build CORTX-ALL image") {
+            steps {
+                script { build_stage = env.STAGE_NAME }
+                script {
+                    try {
+                        def build_cortx_all_image = build job: 'cortx-docker-images', wait: true,
+                                    parameters: [
+                                        string(name: 'CORTX_RE_URL', value: "https://github.com/shailesh-vaidya/cortx-re"),
+                                        string(name: 'CORTX_RE_BRANCH', value: "rocky-linux-custom-ci"),
```

Comment:
Change repository, if possible parameterize it

Reply:
I will raise separate PR once this is merged else it will break job now.